### ISSUE TITLE
figure out sound streaming solution for edmond essay

### ISSUE THREAD

**Issue by elotroalex** [archipelagosjournal/archipelagos#41]
I think we might be able to host the sound files directly on our server, and stream with html5. they don't seem to be that big

**Comment by elotroalex** [comment]
Notes from Kelly:


&lt;ALEX: Because I couldn’t put in hyperlinks to the MP3 files (I don’t
know where those will end up), I put each MP3 file name in brackets
immediately following the text it will link to, which is highlighted in
green.

The author is hoping for embedded audio files rather than a link that
will take readers away from the page. He had originally put in “\[AUDIO
1\]” and so on at each place for a clip, for readers to click on rather
than linked text. But you said earlier that you didn’t want those…. Here
is a note from him: “The whole quotation should be hyperlinked if we go
down the hyperlink route. However, as I’ve discussed with the editors,
it would be preferable to have the extracts embedded within the webpage
so that one isn’t taken away from the text every time one clicks a
link.”

The extracts are numbered, so if you do decide to go back to \[Audio1\]
or some such, the numbers are right there.

I put in footnotes throughout to cite the online locations of all the
digital files, and then Edmonds provided the time codes for the MP3
extracts, so if the piece is printed or dealt with offline, all the
clips are still findable.

No hyperlinks are included, to keep this strictly text and audio
clips.&gt;

**Comment by Denubis** [comment]
We don't want to make a practice of mp3s on github, since the repo limit is 50mb.

Opening a link in a new window is ... trivial? We just need the location of an archive that will mint a DOI for the audio file (and frankly, for all of our old youtube embeds, too). Zenodo won't. https://osf.io/63qsp/?pid=xjphw OSF will, but not sure how to embed timecodes. Do you have any contacts at Harvard's dataverse? (Since we realllllly should make a point of archiving all of our supplemental files?)

**Comment by Denubis** [comment]
Or we could start a major overhaul and move this into LaTeX for https://tex.stackexchange.com/questions/345026/embedded-audio-in-to-pdf -- but this one feature isn't enough to argue for that overhaul.

**Comment by elotroalex** [comment]
Yeah, definitely not putting the mp3s on github. Those are all going on the
.gitignore. I'm still thinking of a solution for storage and one for PDFs.
For videos we were doing images that went on the PDF with the link. Maybe
there's an equivalent design solution here.

Let's open another issue to discuss a major overhaul. The one argument for
that would be more cross-compatibility, imho. Like right now the Docker is
broken for me to use your Linux setup, and I can't replicate it on my Mac
natively. Worth a good discussion probably.
